extensions/lifecycle/bucketProcessor/LifecycleBucketProcessor.js

```diff
@@ -236,7 +236,7 @@ class LifecycleBucketProcessor {
      * @return {Boolean} Whether the config should be processed
      */
     _shouldProcessConfig(config) {
-        if (config.Rules.length === 0) {
+        if (config.Rules.some(rule => rule.Status === 'Enabled')) {
```

Comment:
should we also check that the location associated with the (transition) rule is not paused?